examples/csv-io/csv_io.bal

```diff
@@ -1,6 +1,17 @@
 import ballerina/io;
 import ballerina/log;
 
+type Employee {
+    string id,
+    string name,
+    float salary,
+};
+
+type Salary {
+    string id,
+    float salary,
+};
+
```

Comment:
This sample is assuming user has sample.csv in the ../files location. Instead of that shall we re-organize this to create the .csv file first based on some data( may be using some record data in inmemory table) and then read that .csv file. In that way user can run the sample without having any per-requisists.